main.tf

```diff
@@ -14,65 +14,77 @@
  * limitations under the License.
  */
 
-terraform {
-  required_version = "~> 0.11.0"
-}
-
 locals {
-  prefix = "${var.prefix == "" ? "" : "${var.prefix}-"}${lower(var.location)}"
+  prefix = var.prefix == "" ? "" : join("-", list(var.prefix, ""))
```

Comment:
Is this intentional? The dropping of location looks like a functionality change.

Reply:
True, reasoning was that it's easy to inject that in prefix if needed. But let's revert, move to 0.12, then maybe pick this up again.

Reply:
Agreed, let's do that in a separate PR if it's needed.

Reply:
Reverted and retested, thanks.